client/app/bundles/course/assessment/pages/AssessmentStatistics/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { FC, useEffect, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import { Box, FormControlLabel, Switch, Tab, Tabs } from '@mui/material';
+
+import { fetchAssessmentStatistics } from 'course/assessment/operations/statistics';
+import { statisticsActions as actions } from 'course/assessment/reducers/statistics';
+import Page from 'lib/components/core/layouts/Page';
+import LoadingIndicator from 'lib/components/core/LoadingIndicator';
+import { useAppDispatch, useAppSelector } from 'lib/hooks/store';
+import toast from 'lib/hooks/toast';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import MainGradesChart from './GradeDistribution/MainGradesChart';
+import SubmissionStatusMainAssessment from './SubmissionStatus/MainSubmissionChart';
```

Comment:
Could you be consistent in using the component name?

Reply:
sorry TT it's fixed already

Reply:
sorry :( done fixing the naming